docs/user_guide/assessment/perform_fairness_assessment.rst

```diff
@@ -276,4 +281,46 @@ overall values for the data:
     count    0.222222
     dtype: float64
 
-In every case, the *largest* difference and *smallest* ratio are returned.
\ No newline at end of file
+In every case, the *largest* difference and *smallest* ratio are returned.
+
+
+.. _assessment_predefined_fairness_metrics:
+
+Predefined fairness metrics
+---------------------------
+
+In addition to the disaggregated analysis of base metrics enabled by
+:class:`MetricFrame`, Fairlearn also provides a set of predefined fairness 
+metrics that output a single score. These metrics take as input 
+`sensitive_features` to compute the maximum difference or ratio between 
+subgroups of a sensitive variable. The predefined fairness metrics offered 
+by Fairlearn are **demographic parity** ratio/difference and **equalized odds** 
+ratio/difference. Note that because these metrics are calculated using 
+aggregations between groups, they are meant to be 
+called directly, rather than used within the instantiation of a MetricFrame.
+
+Below, we show an example of calculating demographic parity ratio using the 
+sample data defined above.
+
+.. doctest:: assessment_metrics
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> from fairlearn.metrics import demographic_parity_ratio
+    >>> print(demographic_parity_ratio(y_true,
+    ...                                y_pred,
+    ...                                sensitive_features=sf_data))
+    0.66666...
+
+Note that to achieve a one-number score for a standard performance metric,
+like false positive rate or selection rate, the user must use 
+:func:`make_derived_metric`. Under the hood, the fairness assessment metrics 
+also use :class:`MetricFrame` to compute a particular base rate across 
+sensitive groups and subsequently perform an aggregation (the difference 
+or ratio) on the base metric values across groups. For example, 
+:func:`equalized_odds_ratio` uses both the :func:`false_positive_rate` and
+:func:`false_negative_rate` within a MetricFrame on the backend to generate 
+an output.
+
```

Comment:
Thought.....would it be helpful to add another code block showing how you get to exactly the same result using `MetricFrame` here?

Reply:
Just added an example, but let me know if the language describing the example should change.

Reply:
Perhaps an example which doesn't return '0'? It's great to show how things happen internally, but when both numbers are zero, it's not very convincing (I have a whole separate rant about languages which mix automatic declaration and zero-initialisation).

Reply:
Yeah, I think this would entail generating new data. This recommendation fell off my radar, but I can start mocking something up. If there's synthetic data generated elsewhere in the docs that might also work for this, please let me know.

Reply:
It might be simpler to leave this for now, and file an issue to revise the input data. That's going to propagate all over the place, and isn't central to this PR.

Reply:
Yeah agreed, I will resolve this comment for now and open an issue sometime this week.